.circleci/config.yml

```diff
@@ -3,52 +3,60 @@ orbs:
   anchore: anchore/anchore-engine@1.4.0
 jobs:
   Full Build:
-    machine:
+    machine: &virtual_machine
```

Comment:
✏️ Is this alias needed? Same with `install_java`, `configure_java`, and `restore_cache`.